.travis.yml

```diff
@@ -1,18 +1,29 @@
 language: python
 
+sudo: "required"
+
 python:
-  - "2.7"
-  - "3.5"
+    - "2.7"
+    - "3.5"
 
-sudo: "required"
+env:
+  global:
+    - DISPLAY=:99.0
```

Comment:
Add a matrix using `TOXENV` so we have one job running `py27` and another using `tox`:

```yml
env:
  global:
    - DISPLAY=:99.0
  matrix:
    - TOXENV=flake8
matrix:
  include:
    - python: 2.7
      env: TOXENV=py27
    - python: 3.5
      env: TOXENV=py35
```

You might need to play with this a little, and there could be a better way to get all the jobs to run. Basically you want the following jobs:

 * Python 2.7
    * flake8
    * py27
 * Python 3.5
    * flake8
    * py35

We should also add building the docs to Travis so we get a notification if a pull request brakes these. If it makes it easier, drop python 3.5 from this patch and we'll work on that next.